tests/test_scancode.py

```diff
@@ -43,18 +43,38 @@ def test_init(self):
         scancode = ScanCode(exec_path=SCANCODE_PATH)
         self.assertEqual(scancode.exec_path, SCANCODE_PATH)
 
+        scancode_cli = ScanCode(exec_path=SCANCODE_CLI_PATH)
+        self.assertEqual(scancode_cli.exec_path, SCANCODE_CLI_PATH)
+
         with self.assertRaises(GraalError):
             _ = ScanCode("/tmp/invalid")
 
-    def test_analyze(self):
-        """Test whether nomos returns the expected fields data"""
+    def test_analyze_scancode(self):
+        """Test whether scancode returns the expected fields data"""
 
         scancode = ScanCode(exec_path=SCANCODE_PATH)
         kwargs = {'file_path': os.path.join(self.tmp_data_path, ANALYZER_TEST_FILE)}
         result = scancode.analyze(**kwargs)
 
         self.assertIn('licenses', result)
 
+    def test_analyze_scancode_cli(self):
```

Comment:
The tests look perfect, but it's probably better to separate them in two different classe: TestScanCode and TestScanCodeCli, what do you think ?

Reply:
@valeriocos Thanks for the suggestion.
I had a thought when I worked on this, I proceeded with the thought that they belong to the same analyzer. 
But as you mentioned above that we could (in future) separate both into different analyzers, It seems better to keep two different classes for the test.
I'll update this :)